### ISSUE TITLE
T5 Pretraining and Finetuning for Custom Language

### ISSUE THREAD

**Issue by ashispapu** [huseinzol05/Malaya#52]
Hi,

Thanks for sharing the T5 pre training examples. I have followed the same for another language(Hindi). I have two tasks in the pre training phase(unsupervised and question generation tasks). Then I tried to fine tune the pre trained model for question generation only. However it is not able produce good results on unseen/test data. I have checked the training loss which has reduced significantly and validated on the train data and it does learn that. 

Do you have any suggestions to improve it on fine tuning task.

**Comment by huseinzol05** [comment]
You should ask in original T5 repository, https://github.com/google-research/text-to-text-transfer-transformer/tree/master/t5/models

**Comment by ashispapu** [comment]
Thanks @huseinzol05 for the response . I have asked it there. Wanted to know your pre training data set size(in terms of number of sentences or tokens), so that i might get some idea to debug the issue.
